app/bundles/PointBundle/Model/PointModel.php

```diff
@@ -254,7 +248,7 @@ public function triggerAction($type, $eventDetails = null, $typeId = null, Lead
                     'points'     => $action->getDelta(),
                 ],
                 'lead'         => $lead,
-                'factory'      => $this->factory, // WHAT?
+                'factory'      => '', // WHAT?
```

Comment:
Isn't it used somewhere? It's not that easy to get rid of MauticFactory 😆 

If it the work is out of scope for this issue, please create a new issue instead so we won't forget to refactor it properly.